packages/create-sitecore-jss/src/templates/vue/src/components/Styleguide/Styleguide-Multilingual.vue

```diff
@@ -0,0 +1,46 @@
+<!--
+  Demonstrates using the dictionary functionality and defining route data in
+  multiple languages.
+-->
+<template>
+  <styleguide-specimen v-bind="$props" data-e2e-id="styleguide-multilingual">
+    <sc-text :field="fields.sample" :encode="false" tag="p" />
+
+    <p>
+      <!-- $t is attached to the App instance by the vue-i18n plugin -->
+      This is a static dictionary entry from <code>/data/dictionary</code>:&nbsp;{{
+        $t('styleguide-sample')
+      }}
+    </p>
+
+    <p>
+      <router-link to="/en/styleguide">Show in English</router-link>
+      <br />
+       <router-link to="/<%- language %>/styleguide">Show in <%- language %></router-link>
```

Comment:
If language is not present, you will have undefined here. It should be removed when language doesn't exist

Reply:
Oh yes my bad. We need an ejs conditional around this line.